Habitica/src/main/java/com/habitrpg/android/habitica/ui/helpers/TaskTextParser.kt

```diff
@@ -0,0 +1,51 @@
+package com.habitrpg.android.habitica.ui.helpers
+
+import com.habitrpg.shared.habitica.models.tasks.Task
+import io.reactivex.functions.Consumer
+
+object TaskTextParser {
```

Comment:
I just learned about kotlin extensions. I'll see what I can do with those!
Originally I did this because the object itself couldn't contain any parsing logic due to dependencies. Extensions seem to be the correct solution to that problem rather than what I've done